### PR TITLE
Shalom's Coop blog post

### PR DESCRIPTION
**Pull request by lakikowolfe**
**If your pull request is for adding a blog post, please check that you include the following:**

- [ ] _Title_ - A short, catchy description to entice readers
- [ ] _Author_ - Your name, which should match your contributor [file](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#adding-yourself-as-a-contributor). If you do not have a contributor file, please feel free to contact one of our team members to assist you
- [ ] _Categories_ - Select 1 at minimum of either Community, Technical, or Science as [describe here](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#what-kind-of-posts-are-appropriate-for-our-blog)
- [ ] _Tags_ - Suggested keywords that can help readers find information from among our blog posts. Please suggest a few tags following [our guidelines](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#tags).
- [ ] _Content_ - This is the body of the blog post! We recommend relatively short posts (no more than five paragraphs), equivalent to a three minute read. Pictures and links to other sites are also great!
- [ ] _Reviewer_ - Make sure your post has a tagged reviewer! Select either @k8hertweck or @lakikowolfe for review by clicking the `reviewers` button on the right hand side of the page.

The Coop team member reviewing the submission will check for the following:

- [ ] The post is appropriate for the blog; [see guidelines](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#what-kind-of-posts-are-appropriate-for-our-blog)

- [ ] The author is added as a [contributor](https://fredhutch.github.io/coop/contributors/) to the blog

- [ ] The file is in the DRAFTS folder

- [ ] Link relevant [issues](https://github.com/FredHutch/coop/issues)

- [ ] A plan for publication (either keep in DRAFTS as a backlogged post OR publish immediately by adding yyyy-mm-dd to the title and moving to POSTS folder)

### CODE REVIEW

File: _drafts/TEMPLATE-post-title.md [hunk 1]
```diff
@@ -9,4 +9,12 @@ tags: # add 1-4 lowercase tags that are relevant to your post, ex: r, python, ge
   - r
   - python
 ---
-##Replace this with the content for your blog post
+	In a group full of people, women are 75% less likely to speak up than a male group member. Because chances are their ideas will get rejected or gaslighted by their male peers. Now imagine how much lower that percentage is in a STEM field that’s highly maledominated. In a world where the demand for computer science workers is increasing day by day, then why has the representation of women in this field been declining over the past 30 years? This is where Girls Who Code (GWC) steps in. An organization that grows young female identifying students to join the coding field. Every session the group aims to close the gender ratio gap little by little.
```
Comment (lakikowolfe) [inline _drafts/TEMPLATE-post-title.md]: 1. maledominated -> male-dominated
2. I would rework the third sentence: "Yet, while the demand for computer science workers is increasing, the representation of women in this field has been declining for 30 years."
3. rework the last 2 sentences (they're a little repetitive as is): "GWC is an organization that introduces young female-identifying students to coding. In every session, the group aims to close the gender gap by creating a safe space for girls to code together!

**Comment by lakikowolfe** [comment]
Other notes:
- Please update the name of the file from TEMPLATE to something more descriptive like - shalom-gwc.md
- add a title
- add your name as the author (first and last)
- choose some tags for your post (this is optional)

I love this post! Please make a few edits that I noted above and then we'll be ready to merge!

File: _drafts/TEMPLATE-post-title.md (hunk 1 above)
Comment (Shalom-Abi) [reply _drafts/TEMPLATE-post-title.md]: Thank you!

**Comment by lakikowolfe** [comment]
merged on command line due to conflicts

**Pull request closed without merging**